pjlib/include/pj/config_site_test_full.h

```diff
@@ -0,0 +1,12 @@
+#define PJMEDIA_SRTP_HAS_DTLS 1
+#define PJMEDIA_HAS_WEBRTC_AEC 1
+#define PJMEDIA_CODEC_L16_HAS_8KHZ_MONO 1
+#define PJMEDIA_CODEC_L16_HAS_8KHZ_STEREO 1
+#define PJMEDIA_CODEC_L16_HAS_16KHZ_MONO 1
+#define PJMEDIA_CODEC_L16_HAS_16KHZ_STEREO 1
+#define PJMEDIA_CODEC_L16_HAS_48KHZ_MONO 1
+#define PJMEDIA_CODEC_L16_HAS_48KHZ_STEREO 1
+#define PJMEDIA_HAS_G7221_CODEC 1
+#define PJMEDIA_HAS_G722_CODEC 1
+#define PJ_EXCLUDE_BENCHMARK_TESTS 1
+#define PJ_CLI_MAX_CHOICE_VAL 64
```

Comment:
was thinking to put 64 as PJ_CLI_MAX_CHOICE_VAL default? and I guess it is okay to make it in this same PR.